module/library/categories/helper.js

```diff
@@ -173,6 +174,78 @@ module.exports = class LibraryCategoriesHelper {
 					})
 				}
 
+				let projectTemplates = result[0].data
+				let filePathsArray = []
+				for (let project of projectTemplates) {
+					let categories = project.categories
+
+					let categoryIdArray = categories.map((category) => {
```

Comment:
@borkarsaish65 handle situation if _id is not present

Reply:
done